src/plugins/controls/public/control_group/component/control_frame_component.tsx

```diff
@@ -96,6 +96,7 @@ export const ControlFrame = ({
         'controlFrameFormControlLayout--twoLine': controlStyle === 'twoLine',
       })}
       fullWidth
+      compressed
```

Comment:
These are the old controls. Changes need to be made to controls under the folder `react_controls`. To view the new controls, start kibana with `yarn start --run-examples` and navigate to the controls developer example.

Reply:
Yep! This PR addresses both the old controls and the newer controls.

Reply:
got it, thanks